GOVERNANCE.md

```diff
@@ -84,6 +86,6 @@ In general, if nobody explicitly opposes a proposal, it is recognised as having
 
 ## Changes to this governance policy
 
-Any community member is free to propose changes to this governance policy by submitting a proposed change to this document. The steering committee will discuss any proposed changes and aim to reject or accept the changes by consensus.
+Any community member is free to propose changes to this governance policy by submitting a proposed change to this document, via GH pull request. The steering committee will discuss any proposed changes and aim to reject or accept the changes by consensus.
```

Comment:
Let's actually write out the word GitHub

Reply:
Done